matcher/ironic_config.json

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
I don't think this file should be part of the pull request. I think we will want some tests in this pr, but the tests don't need a full set of data like this. This is mostly useful for development, and it already lives in the flocx documentation repository (https://github.com/CCI-MOC/flocx/).